packs/data/other-effects.db/effect-2-circumstance-penalty-to-checks-and-saving-throws-until-healed.json

```diff
@@ -0,0 +1,68 @@
+{
+    "_id": "c6cwOc4Zri6QiqsR",
+    "img": "systems/pf2e/icons/effects/crtical-hit-fumble.webp",
+    "name": "Effect: -2 circumstance penalty to checks and saving throws until healed",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by Critical Hit/Fumble Deck<br /></strong>-2 circumstance penalty to checks and saving throws until healed</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to saving throws",
+                "selector": "saving-throw",
+                "type": "circumstance",
+                "value": -2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to checks",
+                "selector": "skill-check",
+                "type": "circumstance",
+                "value": -2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to checks",
+                "selector": "perception",
+                "type": "circumstance",
+                "value": -2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to checks",
+                "selector": "attack",
+                "type": "circumstance",
+                "value": -2
+            }
```

Comment:
```suggestion
            {
                "key": "FlatModifier",
                "label": "circumstance penalty to saving throws",
                "selector": "all",
                "type": "circumstance",
                "value": -2
            }
```

Reply:
I think I had broken these up because of the label, but per the previous comment let me look into using more a more generic label and I'll combine these back.